scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -230,6 +237,31 @@ boolean pauseJob(JobId jobId) {
         }
     }
 
+    boolean updateStartAt(JobId jobId, String startAt) {
```

Comment:
Is it possible to put that into a stateless service class? I see that the design of the LiveJobs makes that really hard, but there might be a way?